File: RecycelerViewApp/app/src/main/java/com/example/recycelerviewapp/MainActivity.java
package com.example.recycelerviewapp;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

import java.util.ArrayList;

import Adapators.CartoonAdapators;
import CartoonModel.CartoonModels;

public class MainActivity extends AppCompatActivity {
    RecyclerView recycler;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        recycler =  findViewById(R.id.recyclerView);

        ArrayList<CartoonModels> list = new ArrayList<>();

        list.add(new CartoonModels(R.drawable.a,"Saskea"));
        list.add(new CartoonModels(R.drawable.b,"Teacher"));
        list.add(new CartoonModels(R.drawable.c,"Naruto"));
        list.add(new CartoonModels(R.drawable.d,"Naruto"));
        list.add(new CartoonModels(R.drawable.e,"Itachi"));
        list.add(new CartoonModels(R.drawable.f,"Hacking"));
//        list.add(new CartoonModels(R.drawable.g,"Saskea"));
        list.add(new CartoonModels(R.drawable.h,"Black Widow"));
        list.add(new CartoonModels(R.drawable.i,"Window 11"));
        list.add(new CartoonModels(R.drawable.j,"Man"));
        list.add(new CartoonModels(R.drawable.k,"Binart"));
        list.add(new CartoonModels(R.drawable.l,"Joker"));
        list.add(new CartoonModels(R.drawable.m,"Offline"));

        CartoonAdapators adapators = new CartoonAdapators(list,this);
        recycler.setAdapter(adapators);

        LinearLayoutManager LayoutManager= new LinearLayoutManager(this);
        recycler.setLayoutManager(LayoutManager);
    }
}

File: MyRecycler/app/src/main/java/com/example/myrecycler/MainActivity.kt
package com.example.myrecycler

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        val info = mutableListOf<ContactModel>()
        val recCheck = findViewById<RecyclerView>(R.id.rCView)

        info.add(ContactModel(R.drawable.ma, "Khurram", "52312441"))
        info.add(ContactModel(R.drawable.man, "Usman", "6431411"))
        info.add(ContactModel(R.drawable.mann, "Farhan", "743521"))
        info.add(ContactModel(R.drawable.mannn, "Salman", "8451357"))
        info.add(ContactModel(R.drawable.wom, "Hassan Noor", "1234673"))
        info.add(ContactModel(R.drawable.woman, "Kamran", "175425"))
        info.add(ContactModel(R.drawable.user, "Api", "175425"))

        recCheck.adapter = RecyclerAdaptar(info)
        recCheck.layoutManager = LinearLayoutManager(this)
    }
}